### ISSUE TITLE
CMS > Organisation Content Type

### ISSUE THREAD

**Issue by danlechambre** [grief-matters/grief-matters-cms#112]
In the 'Organisation' content type in the Sanity CMS, we need to ensure that all content for this type is up to date and accurate:

- [ ] Check/update Mission Statement
- [ ] Check/update Slogan
- [ ] Check/update Core Values
- [ ] Check/update Small Print (@danlechambre to be informed is this is the wrong term - should it be 'Legal', 'Disclaimer' etc.)

**Comment by danlechambre** [comment]
Hey @jkraesullivan should these kinds of tasks (relating to content changes within the CMS) be assigned directly to you?

**Comment by jkraesullivan** [comment]
Yes @danlechambre I think it could be either Cherie or I- but likely myself. I think it would make sense to be me, especially after looking at some of these things (I’ve worked a little bit on some of them). It would be easy for me to update the status in GitHub once complete. We can confirm with Camille when we all meet to be sure but for now I think we can assign them to me.
